mappings/net/minecraft/world/entity/EntityIdMapSet.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_5578 net/minecraft/world/entity/EntityIdMapSet
```

Comment:
`MapSet` is very confusing. This is an implementation of the parent interface that stores entities indexed by section, the name should probably clarify that.

Reply:
i hate this class, it's basically just a read-only view of two other data classes

`EntityMapView`? its methods are *mostly* map-like (fetch by id, iterate over values)

Reply:
I'd propose `Sectioned` prefix to the parent interface given its implementation.